.github/workflows/patch-release.yml

```diff
@@ -0,0 +1,94 @@
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+
+name: Patch Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'the version number to release'
+        required: true
+
+env:
+  IMAGE_NAME: aws-otel-collector
+  IMAGE_NAMESPACE: amazon
+  ECR_REPO: aws-observability/aws-otel-collector
+
+jobs:
+  build-image:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build Image
+        run: docker build -t $IMAGE_NAME -f patch/${{ github.event.inputs.version }}/Dockerfile .
```

Comment:
Does this already run from within `tools/release` or is this path wrong?

Reply:
It's the wrong path.